docker/arm.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM arm64v8/alpine:3.16.2
+
+RUN apk add --no-cache gettext curl nginx php8 php8-fpm php8-opcache php8-pdo php8-pdo_sqlite php8-openssl && \
```

Comment:
When bumping to that newer version of Alpine, please change all the packages to be php81.